### ISSUE TITLE
Ternary Operator Error Warning

### ISSUE THREAD

**Issue by Teoreez** [godotengine/godot-vscode-plugin#104]
I'm using Ternary operator as following code:
```
func flip ():
	return true if $Sprite.is_flipped_v() == false else false
```
Godot working with this code easely without warnings or yellow flags.
In VS code happens following Errors warnings:

> ': or \' expected at end of the line //pointing on following code
> Expected indented block after expression //pointing on following code

**Comment by Geequlim** [comment]
The new version fixed the problem and will be published when godot 3.2 RC release.

**Comment by Geequlim** [comment]
I'm closing this as the new version has fixed this issue.
You can try the new version of the plugin with godot 3.2 from the [release page](https://github.com/godotengine/godot-vscode-plugin/releases)
